asset_dashboard/views.py

```diff
@@ -51,19 +53,42 @@ def post(self, request, *args, **kwargs):
 
         if form.is_valid():
             project = Project.objects.create(**form.cleaned_data)
+            project_score = ProjectScore.objects.get_or_create(project=project)
```

Comment:
why is this get_or_create? shouldn't it always be create?

Reply:
> are you actually validating the project scores, or just depending on html5 checks?

That's for the HTML validation.